specs/eventing/spec.md

```diff
@@ -277,6 +277,7 @@ retrieved from ref.
 | `deadLetterSink` | [`duckv1.Destination`](#duckv1.Destination) | Optional    | The sink receiving event that could not be sent to a `Destination`.                                                                                               |             |
 | `retry`          | `int`                                       | Optional    | The minimum number of retries the sender should attempt when sending an event before moving it to the dead letter sink (when specified) or discarded (otherwise). |             |
 | `backoffPolicy`  | `string`                                    | Optional    | The retry backoff policy (`linear` or `exponential`).                                                                                                             |             |
+| `timeout`        | `string`                                    | Optional    | The timeout of each single request.                                                                                                                               |             |
```

Comment:
Note this is a behaviour change for Brokers (all implementations), Triggers, Channels (all implementations), Subscriptions, Sequence, and Parallel.

Reply:
Do we need to say what happens if we hit the timeout? e.g. what should be the error seen?

Reply:
Can we get an RFC keyword here? It’s not clear if this is required to be supported or optional.

Reply:
Is this a timeout per attempt (eg like an http timeout) or the total time you hold on to the event including all retries?